lib/osf-components/addon/components/file-actions-menu/submit-to-boa-modal/component.ts

```diff
@@ -63,6 +62,7 @@ export default class SubmitToBoaModal extends Component<Args> {
                     nodeId: fileModel.target.get('id'),
                     name: file.name,
                     materialized: fileModel.materializedPath,
+                    size: fileModel.size,
```

Comment:
```suggestion
                    sizeInt: fileModel.size,
```

For compatibility, the backend looks for `sizeInt` instead of `size` since for some providers, WB returns `size` with a string instead of an integer.

Reply:
Oh, good catch! I made sure to use the `size` that is a number and not the converted `128MB` string version, and totally forgot the name!